Annotation/constituent_golden_set.json

```diff
@@ -2017,7 +2385,19 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "what it was",
+                "embedded clauses": []
+            },
+            {
+                "predicate": [
+                    {
+                        "str": "was so upset",
+                        "lemma": "",
+                        "POS": ""
+                    }
+                ],
+                "type": "declarative",
+                "clause": "that I just ate it",

```

Comment:
Ambiguous I guess, but I'd interpret it as I was so upset that I was pushed to eating it even though I didn't know what it was (as opposed to being upset at having eaten it)

Reply:
lets discuss later